skbio/stats/distance/_base.py

```diff
@@ -685,6 +689,11 @@ def _is_id_pair(self, index):
                 len(index) == 2 and
                 all(map(lambda e: isinstance(e, string_types), index)))
 
+    def _find_duplicates(self, iterable):
```

Comment:
Awesome, great to hear that!

Thanks!

Yoshiki.

On (Feb-03-15|15:58), Jai Ram Rideout wrote:
> > @@ -685,6 +689,11 @@ def _is_id_pair(self, index):
> >                  len(index) == 2 and
> >                  all(map(lambda e: isinstance(e, string_types), index)))
> >  
> > +    def _find_duplicates(self, iterable):
> 
> Awesome, thanks! I'll port this function over.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/scikit-bio/pull/814/files#r24052198